src/DotVVM.Samples.BasicSamples.AspNetCore/Properties/launchSettings.json

```diff
@@ -12,16 +12,16 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        //"ASPNETCORE_ENVIRONMENT": "Development"
```

Comment:
Did you really mean to comment out the variable?

Reply:
Nop.